docs/java/java-project.md

```diff
@@ -21,7 +21,9 @@ If you run into any issues when using the features below, you can contact us by
 
 Managing a project in VS Code requires the [Project Manager for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension. The extension helps manage class paths and dependencies, and create new projects, packages, and classes.
 
-<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-dependency">Install the Project Manager for Java</a>
+<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-pack">Install the Java Extension Pack</a>
```

Comment:
Do we use the name `Extension Pack for Java` or `Java Extension Pack`?

Reply:
good point - let us use Extension Pack for Java as it is official name on marketplace